common/setups/hybrid/base_system.py

```diff
@@ -0,0 +1,165 @@
+__all__ = ["BaseSystem"]
+
+import itertools
+import sys
+from typing import Dict, List, Optional, Type
+
+# -------------------- Sisyphus --------------------
+
+import sisyphus.global_settings as gs
+import sisyphus.toolkit as tk
+
+# -------------------- Recipes --------------------
+
+import i6_core.am as am
+import i6_core.corpus as corpus_recipes
+import i6_core.meta as meta
+import i6_core.rasr as rasr
+
+from .util import HybridDataInput
+
+# -------------------- Init --------------------
+
+Path = tk.setup_path(__package__)
+
+# -------------------- System --------------------
+
+
+class BaseSystem(meta.System):
+    """
+    - 3 corpora types: train, dev and test
+    - only train corpora will be aligned
+    - dev corpora for tuning
+    - test corpora for final eval
+
+    to create beforehand:
+    - corpora: name and i6_core.meta.system.Corpus
+    - lexicon
+    - lm
+
+    settings needed:
+    - am
+    - lm
+    - lexicon
+    - feature extraction
+    - linear alignement
```

Comment:
some of these settings are not needed in base class but in derived class `GmmSystem`